src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
Thanks.  I'm not suggesting the native work needs to happen now... you can work that out with Manu.  I'm just suggesting that if it does, we should change the managed implementation to account for additional platforms... PlatformNotSupportedException is fine.